groups/bace-iot-platform/measurements/config.yml

```diff
@@ -0,0 +1,5 @@
+version: 1
+name: Measurements
```

Comment:
I would suggest to change the folder to `measurements-webhook` and the name to 'Measurements webhook'